lib/hanami/cli_base.rb

```diff
@@ -0,0 +1,30 @@
+require 'thor'
+
+module Hanami
+  class CliBase < Thor
```

Comment:
@davydovanton It looks like we're using inheritance to share code. For this purpose, a module to include it's better.

What if we turn this into into a module and let the classes to include it?